src/test/java/donnafin/commons/core/types/MoneyTest.java

```diff
@@ -10,49 +10,50 @@
 public class MoneyTest {
 
```

Comment:
Do u wanna comment somewhere here about the appropriate partitions and which test addresses whcih

Reply:
i don't have to, my functions are named clearly enough tho?